src/core/model_config.proto

```diff
@@ -579,6 +579,14 @@ message ModelDynamicBatching
   //@@     batching. Default is 0.
   //@@
   uint64 max_queue_delay_microseconds = 2;
+
+  //@@  .. cpp:var:: bool preserve_ordering
+  //@@
+  //@@     If true, the responses will be returned in the same order as the
+  //@@     order of requests sent to the scheduler. Otherwise the responses will
```

Comment:
"If false the responses [may] be returned in an arbitrary order. This option is specifically needed when a sequence of related inference requests (i.e. inference requests with the same correlation ID) are sent to the dynamic batcher to ensure that the sequence responses are in the correct order."